xhr.bs

```diff
@@ -916,18 +920,17 @@ return <a>this</a>'s <a>cross-origin credentials</a>.
   <ol>
    <li><p>If <a>this</a>'s <a>relevant settings object</a> has a <a>responsible document</a> which
    is not <a>allowed to use</a> the "<code><a>sync-xhr</a></code>" feature, then run
-   <a>handle response end-of-body</a> for <a>this</a> and a <a>network error</a>, and then return.
-
-   <li><p>Let <var>response</var> be a <a for=/>network error</a>.
+   <a>handle response end-of-body</a> for <a>this</a> and return.
 
    <li><p>Let <var>processedResponse</var> be false.
 
    <li>
-    <p>Let <var>processResponseEndOfBody</var>, given a <var>fetchResponse</var> and
+    <p>Let <var>processResponseEndOfBody</var>, given a <var>response</var> and
     <var>nullOrFailureOrBytes</var>, be these steps:
 
     <ol>
-     <li><p>Set <var>response</var> to <var>fetchResponse</var>.
+     <li><p>If <var>nullOrFailureOrBytes</var> is not failure, then set <a>this</a>'s
```

Comment:
I think we also need

if _nullOrFailureOrBytes_ is failure, then set **this**'s response to a network error.

Reply:
This is a bit bad perhaps, but the default value of this's response is a network error. (This mainly makes it easier to define a lot of the getters.)